docs/ops.html

```diff
@@ -3800,9 +3800,9 @@ <h4 class="anchor-heading"><a id="kraft_missing" class="anchor-link"></a><a href
   <h4 class="anchor-heading"><a id="kraft_zk_migration" class="anchor-link"></a><a href="#kraft_zk_migration">ZooKeeper to KRaft Migration</a></h4>
 
   <p>
-    <b>ZooKeeper to KRaft migration is considered an Early Access feature and is not recommended for production clusters.</b>
-    Please report issues with ZooKeeper to KRaft migration using the
-    <a href="https://issues.apache.org/jira/projects/KAFKA" target="_blank">project JIRA</a> and the "kraft" component.
+    <b>ZooKeeper to KRaft migration is ready for production clusters since version 3.6</b>
+    Please refer to
+    <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-833%3A+Mark+KRaft+as+Production+Ready#KIP833:MarkKRaftasProductionReady-Kafka3.5" target="_blank">KIP-833</a> for more information.
```

Comment:
I think we can just remove all this paragraph, like in other sections, we won't mention anything about "ready for production". WDYT?